hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/security/SecurityUtil.java

```diff
@@ -513,6 +515,7 @@ public static Text buildTokenService(URI uri) {
    * @param <T> generic type T.
    * @return generic type T.
    */
+  @Deprecated
```

Comment:
let's mention the suggested replacement when marking deprecated

Reply:
Thanks, added.